Datahub/src/Datahub.Portal/Views/Modals/CreateProjectDialog.razor

```diff
@@ -54,15 +64,20 @@
     private MudTextField<string>? _acronymField;
 
     private bool _success;
-    
+    private bool _isCreating;
+
+    protected override void OnInitialized()
+    {
+        MudDialog.SetTitle(Localizer[Title]);
```

Comment:
Ah, here's where it's setting. Are we able to set this when we show dialog instead?

Reply:
In the updated commits, I changed where the dialog title is set,